ui/app/styles/components/toolbar.scss

```diff
@@ -106,3 +108,7 @@
   margin: 0 $spacing-xs;
   width: 0;
 }
+
+.toolbar-button-spacer {
+  margin-left: 4px; // Note: We should give a reco what spacing we want, when inline with text
```

Comment:
wanted to crosspost from slack convo to explain what i meant:

- the structure-icons included some padding as part of the .svg file. the flight icons do not include padding within the .svg
- i added 4px horizontal spacing between the default 16 size flight icon and adjacent inline text, which i think looks fine
- eventually we should bring those recommendations to the documentation of flight https://github.com/hashicorp/flight (link in the upper RHS, just sharing the repo in case the vercel link changes)